packages/perseus/src/widgets/interactive-graphs/backgrounds/axis-ticks.tsx

```diff
@@ -4,41 +4,30 @@ import {useTransformVectorsToPixels} from "../graphs/use-transform";
 import {MAX, MIN, X, Y} from "../math";
 import useGraphConfig from "../reducer/use-graph-config";
 
-import type {GraphDimensions} from "../types";
-import type {vec} from "mafs";
+import type {Interval, vec} from "mafs";
 
+// The size of the ticks and labels in pixels
 const tickSize = 10;
+const tickLabelSize = 14;
 
```

Comment:
The font size is important for calculating positioning, so we're handling it within the component.